django/core/management/commands/squashmigrations.py

```diff
@@ -75,7 +74,7 @@ def handle(self, **options):
             raise CommandError(str(err))
         # Load the current graph state, check the app and migration they asked
         # for exists.
-        loader = MigrationLoader(connections[DEFAULT_DB_ALIAS])
+        loader = MigrationLoader(None)
```

Comment:
connection is used to check which migrations have been applied on the system that runs the command. But we don't want generation of the squash migration to depend on the applied status.